### DISCUSSION TITLE
Error Encountered When Running the Workflow Diagram Example Server Standalone JAR

### DISCUSSION THREAD

**Discussion by Mohammad-Si** [eclipse-glsp/glsp#1291] [Q&A]
I have pulled the latest version of the GLSP server and successfully built it. However, upon attempting to run it, I encountered an error:
<img width="438" alt="image" src="https://github.com/eclipse-glsp/glsp/assets/72154633/0af2db31-dabc-46b5-b35d-814658a0136f">

**Comment by tortmayr** [comment]
Hi @Mohammad-Si ,
Looks like you are trying to start the wrong jar.
In order to run the GLSP from a jar you have to create a `fatjar` (that includes all dependencies)
with `mvn clean verify -Pm2 -Pfatjar` and then start the created fatjar (suffixed with -glsp).
You can find detailed instructions in the [README](https://github.com/eclipse-glsp/glsp-server?tab=readme-ov-file#building-the-workflow-diagram-example-server).

Please note that the example in the `glsp-server` repo is mainly a development example and is not the ideal starting point.
If you want to start with the implementation of your own GLSP editor I recommend to checkout one of the projecttemplates in the `eclipse-glsp-examples` repository (https://eclipse.dev/glsp/documentation/gettingstarted/)

**Reply by Mohammad-Si** [reply]
Hi @tortmayr 

I had done all the instructions in the [README](https://github.com/eclipse-glsp/glsp-server?tab=readme-ov-file#building-the-workflow-diagram-example-server), and also, I run the "org.eclipse.glsp.example.workflow-2.1.0-glsp.jar" file.
<img width="532" alt="image" src="https://github.com/eclipse-glsp/glsp/assets/72154633/6dff641b-8677-4b8a-89c6-2c41935f0673">

**Reply by tortmayr** [reply]
I see, could you please provide us with more information about your system, so that we can try to reproduce the error`
OS, Maven version and Java version.

**Reply by Mohammad-Si** [reply]
@tortmayr 

OS: Windows 11 Pro,
Java: 17.0.10,
Maven: 3.8.8

Thank you. Actually, I just added the below dependency to the [POM](https://github.com/eclipse-glsp/glsp-server/blob/master/examples/org.eclipse.glsp.example.workflow/pom.xml), and then the server successfully came up:
"2024-03-25 11:00:10,718004800 [main] INFO  SocketGLSPServerLauncher - The GLSP server is ready to accept new client requests on port: 5007
[GLSP-Server]:Startup completed. Accepting requests on port:5007"

 However, if something goes wrong with configuring it on my desktop, I would appreciate your help with that.

**Reply by madamedefarge** [reply]
@Mohammad-Si 
Please can you detail which dependency you added to the POM as I have the same issue trying to run the jar.
